src/library/f3d_options.cxx

```diff
@@ -155,6 +155,7 @@ options::~options() = default;
 
 //----------------------------------------------------------------------------
 options::options(const options& opt)
+  : Internals(new options::F3DInternals)
```

Comment:
This is not tested anymore, but it actually should not have been before. I keep the method for now.